Library/Formula/gitlab-ci-multi-runner.rb

```diff
@@ -31,10 +24,12 @@ def install
       # Copy from Makefile
       system "go", "build", "-o", "gitlab-ci-multi-runner", "-ldflags", "-X main.NAME=gitlab-ci-multi-runner -X main.VERSION=#{version} -X main.REVISION=#{commit_sha}"
       bin.install "gitlab-ci-multi-runner"
+      bin.install_symlink "gitlab-ci-multi-runner" => "gitlab-runner"
```

Comment:
There’s an error because `bin.install "gitlab-ci-multi-runner"` *moves* the executable under `bin/`. To fix it use the following:
```rb
bin.install_symlink "#{bin}/gitlab-ci-multi-runner" => "gitlab-runner"
```